free-programming-books-pt_PT.md

```diff
@@ -32,3 +33,8 @@
 ### Prolog
 
 * [Lógica Computacional (com Prolog)](http://www4.di.uminho.pt/~mjf/pub/LC-Prolog.pdf) - Maria João Frade (PDF)
+
+
+### Python
+
+* [Python Para Todos: Explorando Dados com Python 3](http://do1.dr-chuck.com/pythonlearn/PT_br/pythonlearn.pdf) - Dr. Charles Russell Severance (PDF) [(EPUB)](http://do1.dr-chuck.com/pythonlearn/PT_br/pythonlearn.epub)
```

Comment:
Missing new line at the end of this resource.

> 1 empty line at the end of each .md file

as per the contributing guidelines.

Reply:
Thanks!!!